model/src/main/resources/contract/AssetModuleService.wsdl

```diff
@@ -287,9 +289,71 @@
                 </xsd:complexType>
             </xsd:element>
 
-        </xsd:schema>
+            <xsd:complexType name="VesselIdentifiersHolder">
```

Comment:
Keep only request/response types here, move the rest to their respective domain XSD. Move this type to `AssetElement.xsd`.